drivers/ethernet/dsa/dsa_nxp_imx_netc.c

```diff
@@ -25,6 +26,10 @@ LOG_MODULE_REGISTER(dsa_netc, CONFIG_ETHERNET_LOG_LEVEL);
 struct dsa_netc_port_config {
 	const struct pinctrl_dev_config *pincfg;
 	netc_hw_mii_mode_t phy_mode;
+#ifdef CONFIG_NET_QBV
+	netc_tb_tgs_gcl_t tgs_config;
+	netc_tgs_gate_entry_t gcList[CONFIG_DSA_NXP_NETC_GCL_LEN];
+#endif
```

Comment:
data structure should be used for runtime data. Unfortunately, data of all ports points to common dsa_switch_context.
So, I think we could implement code as below. Finding the config could be by dsa_port_config.port_idx.

```
struct netc_qbv_config {
    netc_tb_tgs_gcl_t tgs_config;
    netc_tgs_gate_entry_t gcList[CONFIG_DSA_NXP_NETC_GCL_LEN];
}

struct dsa_netc_data {
    ...
    struct netc_qbv_config qbv_config[DSA_PORT_MAX_COUNT];
    ...
}
```